index.html

```diff
@@ -1,48 +1,316 @@
+                
+                
+
+
+
+            </section>
+        </div>
+    </div>
+        
+    
+        <section id="aboutme" class="container9">
+
+                <div class="about item9"><h1>About me</h1> </div>
+
+                <div class="description item9"><p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have
+                    a project you need coded, don’t hestiate to contact me. </p>
+                </div>
+
+                <div class="item9">
+                    <button class="resume">My Resume</button>
+                </div>
+
+                <section class="item9 coding-languages">
+
+                    <div><img class="lang-icon" alt="lang icon" src="img/icon-languages.png"></div>
+
+                    <div><h2 class="res">Languages</h2></div>
+
+                    <div>
+                    <ul id="btn7">
+                        <li><button class="technologies"> JavaScript</button></li>
+                        <li><button class="technologies"> Ruby</button></li>
+                        <li> <button class="technologies"> HTML</button></li>
+                        <li><button class="technologies"> CSS</button></li>
 
+                    </ul>
+                    </div>
+
+                </section>
+
+                <section class="item9 frameworks">
+                    <div><img class="lang-icon " alt="lang icon" src="img/icon-frameworks.png"></div>
+
+                    <div><h2 class="res">frameworks</h2></div>
+
+                    <div>
+                        <ul id="btn8">
+                        <li><button class="technologies"> Bootstrap</button></li>
+                        <li><button class="technologies"> Ruby on rails</button></li>
+                        <li><button class="technologies"> RSpec</button></li>
+                         <li><button class="technologies"> Capybara</button></li>
+                        <li> <button class="technologies"> Selenium</button></li>
+                    </ul>
+                    </div>
+
+                </section>
+                
+                <section class="item9 skills">
+
+                    <div><img class="lang-icon item9-3" alt="lang icon" src="img/icon-skills.png"></div>
+
+                    <div><h2 class="res">skills</h2></div>
+
+                    <div>
+                    <ul id="btn9">
+                        <li><button class="technologies"> Codekit</button></li>
+                        <li><button class="technologies"> GitHub</button></li>
+                        <li><button class="technologies"> Codepen</button></li>
+                        <li><button class="technologies"> Gitlab</button></li>
+                        <li><button class="technologies"> Terminal</button></li>
+                    </ul>
+                    </div>
+
+                </section>
+            
+        </section>
```

Comment:
- [ ] **REQUIRED:** This section should be aligned the same with the `My recent works` section, but it is flowing to the right side, use `inspector` to check it, and please give the same align just by giving the same classes.

![image](https://user-images.githubusercontent.com/77872355/129403278-2567e2a9-6edf-4749-ad85-e9cf599ab549.png)